src/app/content/components/Page/contentLinkHandler.ts

```diff
@@ -138,7 +138,7 @@ export const contentLinkHandler = (anchor: HTMLAnchorElement, getProps: () => Co
 
     e.preventDefault();
 
-    if (isHtmlElementWithHighlight(e.target)) {
+    if (isHtmlElementWithHighlight(e.target) && hash !== reference?.match) {
```

Comment:
i think this line was added here https://github.com/openstax/rex-web/pull/485/files#diff-3f2d87fbf043fd71538b475ed64199e922160643163bab450faa5f753dd9024aR73 as a way to not clear the highlight editing when clicking figure links. as you can see its supposed to allow clicks through when they're the focused highlight but i don't seem to be able to click any links inside highlights so i think something has gotten broken along the way.

Reply:
This change did make the click work on Figure 1.7 for me.
You can try it in the [deployment (with a query already entered)](https://rex-web-fix-search-high-e3k7v6.herokuapp.com/books/american-government-2e/pages/1-2-who-governs-elitism-pluralism-and-tradeoffs?query=figure%201.7&target=%7B%22type%22%3A%22search%22%2C%22index%22%3A0%7D#fs-id1171472068209)

Reply:
or do we just need another card

Reply:
no, i don't think the fix here is correct. it only addresses hash links, [other search highlights over other content links](https://rex-web-fix-search-high-e3k7v6.herokuapp.com/books/american-government-2e/pages/2-1-the-pre-revolutionary-period-and-the-roots-of-the-american-political-tradition?query=Thomas%20Jefferson%2C%20the%20Declaration%20of%20Independence%20officially&target=%7B%22type%22%3A%22search%22%2C%22index%22%3A0%7D#fs-id1166226347619) will continue to not work. we need a more generic solution for allowing clicks to go through when clicking in highlights.

i was saying that previously we tried to make it so clicking in a user highlight would first focus it, and then allow you to click the link with the second click. this appears to be broken now, and in any case would not have worked with search highlights because its specifically looking for the focused user highlight. 

maybe we can do a more dom-based check by looking for the focused class on the highlight

in any case i think there is more work to do on this issue to find a good solution, preferably one that works for both user and search highlgihts